pkg/kubelet/container_manager_unsupported_test.go

```diff
@@ -0,0 +1,72 @@
+// +build !linux
```

Comment:
After this merges and I rebase, I can do the refactor in https://github.com/kubernetes/kubernetes/pull/14675

There's a common fake for reuse between platforms.